config/apply/ccsm/collapse.js

```diff
@@ -35,12 +37,15 @@ export function collapseIntoOne(cards, useHtml=false) {
       riskTable
     } = details;
     let sources = getSources(decisionAids[0]?.extension?.documentation.filter(d => d.label === recommendationGroup));
-    console.log('Sources:');
-    console.log(sources);
     // Generate the markdown details
+    // The first item of the recommendation details array is the 'main' part of the recommendation; display in bold
+    let mainRecommendation = '**' + recommendationDetails[0] + '**';
     let markdown =
-      '# ' + recommendation + ' (' + recommendationGroup + ') ' + '\n\n' +
-      recommendationDetails.join('\n\n') + '\n\n';
+      '# ' + recommendation + '\n\n' +
+      mainRecommendation + '\n\n' +
+      recommendationDetails.slice(1).join('\n\n') + '\n\n';
+    group = "**Reference**: " + recommendationGroup + "\n";
```

Comment:
eslint doesn't like the double quotes used in this line.
```suggestion
    group = '**Reference**: ' + recommendationGroup + '\n';
```